sips/pending/_posts/2014-06-27-42.type.md

```diff
@@ -2,6 +2,9 @@
 layout: sip
 disqus: true
 title: SIP-23 - Literal-based singleton types
+
+vote-status: dormant
+vote-text: This proposal is dormant because original authors are not available for its review. There is an implementation <a href="https://github.com/scala/scala/pull/5310">here</a>.
```

Comment:
Did you test this locally? I was under the impression that links don't work in front-matter.